Source/WebKit/NetworkProcess/cocoa/NetworkDataTaskCocoa.mm

```diff
@@ -325,7 +327,7 @@ static float toNSURLSessionTaskPriority(WebCore::ResourceLoadPriority priority)
     setCookieTransform(request, IsRedirect::No);
     if (shouldBlockCookies(thirdPartyCookieBlockingDecision)) {
 #if !RELEASE_LOG_DISABLED
-        if (m_session->shouldLogCookieInformation())
```

Comment:
It looks like you can also mark m_session const, though this won't help with the warnings as it holds a WeakPtr.